test/integration/targets/connection_aws_ssm/aliases

```diff
@@ -0,0 +1,4 @@
+unsupported
```

Comment:
I'd like to see this test be run in CI instead of being marked `unsupported`.

What is blocking this from running in CI?

Reply:
@gau1991 Doesn't https://github.com/ansible/ansible/commit/8f6b231c41fa6f2b607f717a3eafe4cd5b77bc66 allow this to be removed now?

Reply:
@mattclay Does this need something [like](https://github.com/ansible/ansible/blob/3d19cf79cf7c44e1206b3b58959ec520010d021e/test/integration/targets/aws_s3/aliases#L2) `shippable/aws/group1` added now to get past the [failure](https://app.shippable.com/github/ansible/ansible/runs/114805/1/tests)?

Reply:
Yes, `shippable/aws/group1` is fine to start with.